docs/tutorials/amd-igpu-780m.md

```diff
@@ -0,0 +1,116 @@
+
+The iGPU 780M of AMD Ryzen CPU could run on ROCm at Linux with extral settings. 
+
+Available CPU: Ryzen 7000s/8000s CPU with iGPU 780M
+
+## Test Platform
+
+| **Platform ** | **miniPC**                                                      |
+| ------------- | --------------------------------------------------------------- |
+| **HW**        | AMD Ryzen R8845HS + Radeon780M(iGPU, set 16GB VRAM of 64GB DDR) |
+| **OS**        | Ubuntu22.04                                                     |
+| **SW**        | ROCm6.0+PyTorch                                                 |
+
+
+## Prerequisites
+0. Set UMA for iGPU in BIOS.
+1. Install GPU Driver and ROCm
+	Refer to 
+	 - [AMD ROCm™ documentation — ROCm Documentation](https://rocmdocs.amd.com/en/latest/)
+	 - [Ubuntu native installation — ROCm installation (Linux) (amd.com)](https://rocm.docs.amd.com/projects/install-on-linux/en/latest/how-to/native-install/ubuntu.html) 
+	Steps:
+		Install AMD GPU Driver
+		`sudo apt install amdgpu-dkms`
+		`sudo reboot`
+		Install ROCm stack
+		`sudo apt install rocm`
+
+2. Install PyTorch-ROCm6.0
+	I suggest to use conda to manage your environment
+	`pip3 install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/rocm6.0`
```

Comment:
Can you clarify why this is required?  (I wouldn't think it would be)

Reply:
Yes. For ollama, it is not need. I will remove this information.